cocotb/share/makefiles/simulators/Makefile.questa

```diff
@@ -42,18 +42,18 @@ ifeq (, $(CMD))
     $(error "Unable to locate command >$(CMD_BIN)<")
 else
     MODELSIM_BIN_DIR := $(shell dirname $(CMD))
```

Comment:
Is this line used?

Reply:
Not used. Was used in the past. -> Removed